projects/media-viewer/src/lib/icp/participants-list/participants-list.component.ts

```diff
@@ -0,0 +1,38 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { Observable, Subscription } from 'rxjs';
+import { ToolbarEventService } from '../../toolbar/toolbar-event.service';
+import { IcpParticipant, IcpState } from '../icp.interfaces';
+import { select, Store } from '@ngrx/store';
+import * as fromSelectors from '../../store/selectors/icp.selectors';
+
+@Component({
+  selector: 'mv-participants-list',
+  templateUrl: './participants-list.component.html'
+})
+export class ParticipantsListComponent implements OnInit, OnDestroy {
+
+  subscription: Subscription;
+  participants$: Observable<IcpParticipant[]>;
+  presenter$: Observable<IcpParticipant>;
+  isPresenter$: Observable<boolean>;
+
+  showParticipantsList = false;
+
+  constructor(private readonly toolbarEvents: ToolbarEventService,
+              private store: Store<IcpState>) {}
+
+  ngOnInit() {
+    this.participants$ = this.store.pipe(select(fromSelectors.getParticipants));
+    this.presenter$ = this.store.pipe(select(fromSelectors.getPresenter));
+    this.isPresenter$ = this.store.pipe(select(fromSelectors.isPresenter));
+
+    this.subscription = this.toolbarEvents.icp.icpParticipantsListVisible.subscribe(isVisible => this.showParticipantsList = isVisible);
+    this.subscription.add(this.toolbarEvents.commentsPanelVisible.subscribe(isVisible => {
```

Comment:
I think the mutual exclusivity of comments-panel and participants list should be managed at the toolbar-events service level, and rather than the components